contrib/chart/backstage/Chart.yaml

```diff
@@ -0,0 +1,30 @@
+apiVersion: v2
+name: backstage
+description: A Helm chart for Spotify Backstage
+type: application
+
+# This is the chart version. This version number should be incremented each time you make changes
+# to the chart and its templates, including the app version.
+version: 0.1.0
+
+# This is the version number of the application being deployed. This version number should be
+# incremented each time you make changes to the application.
+appVersion: v0.1.1-alpha.23
+
+sources:
+  - https://github.com/spotify/backstage
+  - https://github.com/spotify/lighthouse-audit-service
+
+dependencies:
+  - name: postgresql
+    condition: postgresql.enabled
+    version: 9.4.1
+    repository: https://charts.bitnami.com/bitnami
+
+maintainers:
+  - name: Martina Iglesias Fernández
```

Comment:
Should we add the core team here?

Reply:
I'm thinking probably not right now, as most stuff in the contrib folder is maintained by contributors